gluefactory/models/extractors/jpldd/line_detection_lsd.py

```diff
@@ -0,0 +1,37 @@
+from gluefactory.geometry.line_utils import merge_lines, filter_outlier_lines
+from gluefactory.utils.deeplsd_utils import preprocess_angle
+from pytlsd import lsd
+import numpy as np
+
+
+def detect_afm_lines(img: np.array, df: np.array, line_level: np.array, filtering='normal',
```

Comment:
Is this taken off the shelf from lsd? There are some constants I wonder where they are coming from. (ex gradnorm = np.maximum(5 - df, 0).astype(np.float64))

Reply:
This is taken from deeplsd_inference, yeah. It's practically the same function